SEED-platform/seed
# Add helper text to Audit Template settings

#### Any background context you want to provide?
Adding helper text.

#### What's this PR do?
References the "AUDIT_TEMPLATE_HOST" in settings to bring in the base url for Audit Template API page.

#### How should this be manually tested?
Go to an Organization's settings page and click on "API Key". Check that the Audit Template fields have helper text and make sure the links work as intended.

#### What are the relevant tickets?
Fixes #3374

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/6326294/181001292-c10ea8b1-6273-41c8-8741-ccdfb344c718.png)
